doc/en/components/grids/_shared/column-resizing.md

```diff
@@ -226,14 +226,14 @@ public onResize(event) {
 <!-- end: WebComponents -->
 
 ```tsx
-function onResize(grid: IgrGridBaseDirective, event: IgrColumnResizeEventArgs) {
-  IgrColumn col = event.detail.column;
-  string pWidth = event.detail.prevWidth;
-  string nWidth = event.detail.newWidth;
+const onResize = (event: IgrColumnResizeEventArgs) => {
+  const col = event.detail.column;
+  const pWidth = event.detail.prevWidth;
+  const nWidth = event.detail.newWidth;
 }
 
-<{ComponentSelector} id="hierarchicalGrid" autoGenerate="false" columnResized={onResize}>
-    <IgrColumn field="Artist" resizable="true"></IgrColumn>
+<{ComponentSelector} id="hierarchicalGrid" autoGenerate={false} columnResized={onResize}>
```

Comment:
There are couple of events that should be prefixed with `oN` in hGrid and tree grid